experiences/homepage/homepage.test.js

```diff
@@ -52,6 +52,54 @@ const homepageTest = function() {
 		expect(cardExists).equal(true);
 	});
 
+	it('scroll bounded', async function() {
+		//TODO implement utility to get current SnapPoint
+		let currentSnapPoint = 623;
+		// First 60px transitioner doesn't made the card appear
+		const pixelsWithoutAnimation = 60;
+		const smallScrollPixels = 20;
+		// Transform to pixels
+		// TODO to calculate relative factor
+		const scrollBoundFactor = 2.1;
```

Comment:
Not sure which is the formula to get this factor. Pending to test it with different screen sizes too. Any theory on how to calculate this? @davidmohringmrf